WTWSMS/common/landed_titles/mercenaries.txt

```diff
@@ -2540,7 +2543,14 @@ d_ghanan_band = {
 	
 	# Hire Trigger
 	allow = {
-		culture_group = west_african
+		OR = {
+			culture_group = west_african
+			culture_group = central_african
+			culture = manden
+			culture = berber
+			religion = west_african_pagan
+			religion = west_african_pagan_reformed
+		}
 	}
```

Comment:
Why? You already made the Valentianian Holy Order out of the Sunni Berber Company.